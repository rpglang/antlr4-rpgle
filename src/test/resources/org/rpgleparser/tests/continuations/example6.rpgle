       /FREE
                  time = hours * num_employees
                           + overtime_saved;
       /END-FREE