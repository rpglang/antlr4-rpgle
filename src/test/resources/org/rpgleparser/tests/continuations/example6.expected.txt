/*===TOKENS===*/
DIRECTIVE                <  />
DIR_FREE                 <FREE>
EOL                      <>
TIME                     <time>
'='                      <=>
ID                       <hours>
MULT                     <*>
ID                       <num_employees>
'+'                      <+>
ID                       <overtime_saved>
FREE_SEMI                <;>
DIRECTIVE                <  />
DIR_ENDFREE              <END-FREE>
/*===TREE===*/
(r
  (statement (directive '  /' (beginfree_directive FREE) ))
  (statement
    (free
      (baseExpression
        (op
          (op_eval
            (evalExpression
              (assignmentExpression
                (simpleExpression
                  (identifier (free_identifier (idOrKeyword (datatypeName time))))
                )
                =
                (expression
                  (expression
                    (expression (identifier (free_identifier (idOrKeyword hours))))
                    *
                    (expression (identifier (free_identifier (idOrKeyword num_employees))))
                  )
                  +
                  (expression (identifier (free_identifier (idOrKeyword overtime_saved))))
                )
              )
            )
          )
        )
      )
      ;
    )
  )
  (statement (directive '  /' (endfree_directive END-FREE) <EOF>))
)
/*======*/